plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -517,3 +524,32 @@ def dcnm_get_url(module, fabric, path, items, module_name):
         iter += 1
 
     return attach_objects
+
+
+def dcnm_get_protocol_and_address(module):
```

Comment:
Can you add some comments on the purpose of these 3 new functions?  Also why do we need the new dcnm_post_request method?

Reply:
ok